packages/Core/theme/modals.ts

```diff
@@ -38,9 +39,10 @@ export const getModals = (theme: WuiTheme): ThemeModals => {
     },
     content: {
       color: colors['dark-900'],
+      paddingTop: space.xxl,
+      paddingRight: space['3xl'],
```

Comment:
Was this validated with the design? Because we have cases where we don't have an header but where it still need to be symmetrical

Reply:
Yes, it is actually just the same code as before, but moved in the theme instead of being hardcoded (https://github.com/WTTJ/welcome-ui/pull/1750/files#diff-8a4e9c41140cab35d9946dc02af2c744daf2da61d3bea530633032863cae8b38L100-L102)